src/custom/pages/Faq/index.tsx

```diff
@@ -173,14 +173,21 @@ export default function Faq() {
             features while placing gas free trades.
           </p>
 
-          <h3 id="why-is-cowswap-a-meta-dex-aggregator">What makes CowSwap a "Meta" DEX aggregator?</h3>
-          <p>
-            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions for a variety of on-chain liquidity sources.
-            Trades can be settled via underlying on-chain AMMs directly or via DEX Aggregators, depending on which pool/path offers 
-            the best price. It is thus essentially acting as a DexAggregator of the DexAggregators.  In addition to that, before 
-            finding the best price for a trade from available on-chain liquidity, Gnosis Protocol first seeks a coincidence of wants within the existing batch to offer an even better price than any pool can.
-          </p>          
-          
+          <h3 id="why-is-cowswap-a-meta-dex-aggregator">What makes CowSwap a &quot;Meta&quot; DEX aggregator?</h3>
+          <p>
+            Cowswap is built on top of the Gnosis protocol which matches trades via batch auctions for a variety of
+            on-chain liquidity sources.
```

Comment:
Could we please put sentences on their own line!

Reply:
What do you mean Ben? I've split them into paragraphs already
![screenshot_2021-06-10_07-24-35](https://user-images.githubusercontent.com/43217/121542280-e2bcf100-c9bc-11eb-986c-f243c12a6fe9.png)

Except the second paragraph, because in my mind it makes sense to have both phrases on the same.

Reply:
I mean that in the file each sentence should be on its own line. So that suggestions can be made to a single sentence and it is easier to parse the diff

Reply:
I see. In that case, that's on the linter.
#wontfix ;)